frontend/src/components/transactions/ActionRowBlock.tsx

```diff
@@ -162,7 +162,7 @@ const ActionRowBlock: React.FC<Props> = React.memo(
           </Col>
           <ActionRowDetails type={viewMode}>
             <ActionRowMessage type={viewMode} noGutters>
-              <Col md="8" xs="7">
+              <Col md={detalizationMode === "detailed" ? 8 : 12} xs="7">
```

Comment:
This is how it looks like on staging:
<img width="527" alt="Снимок экрана 2022-05-27 в 13 04 26" src="https://user-images.githubusercontent.com/34593263/170678748-b059f02e-eaad-40ae-9d82-8f387ff262bb.png">

Maybe it's better to make it full width on mobile?
```suggestion
              <Col md={detalizationMode === "detailed" ? 8 : 12} xs="12">
```

Reply:
It should depend on `detalizationMode` as well.